presto-docs/src/main/sphinx/connector/iceberg.rst

```diff
@@ -491,14 +491,14 @@ JMX queries to get the metrics and verify the cache usage::
 
     SELECT * FROM jmx.current."com.facebook.presto.hive:name=iceberg_parquetmetadata,type=cachestatsmbean";
 
-Metastore Versioned Cache
-^^^^^^^^^^^^^^^^^^^^^^^^^
+Metastore Cache
+^^^^^^^^^^^^^^^
 
 Metastore cache only caches schema and table names. Other metadata would be fetched from the filesystem.
 
 .. note::
 
-    Metastore Versioned Cache would be applicable only for Hive Catalog in the Presto Iceberg connector.
+    Metastore Cache would be applicable only for Hive Catalog in the Presto Iceberg connector.
```

Comment:
I'm a little confused here, have we mixed the cache in `IcebergAbstractMetadata` with the cache in `InCacheCachingHiveMetastore` in this description? The cache that only caches schema and table names is in `IcebergAbstractMetadata`, and it is applicable for all kind of catalogs. While the cache that is  applicable only for Hive catalog is in `InCacheCachingHiveMetastore`, and it can also cache other kind of metadata instances such as `Table` and `PartitionStatistics` etc. Any misunderstanding please let me know. cc: @agrawalreetika

Reply:
I don't think it was confused with the cache in IcebergAbstractMetadata. It was documented for the cache in InMemoryCachingHiveMetastore itself but it can be rephrased. 

When using the Hive catalog with Iceberg Connector, I believe only the following caches will be used from InMemoryCachingHiveMetastore: `databaseCache`, `databaseNamesCache`, `tableCache`, `tableNamesCache`, `tableStatisticsCache`. The usage of table cache is limited since the table object cached in the `tableCache` is only used for reading the table metadata location and table properties and, the rest of the table metadata is fetched from the metadata location.

So, we can probably rephrase the original documentation as:

```
Metastore Cache only caches the schema, table, and table statistics. The table object cached in the `tableCache` is only used for reading the table metadata location and table properties and, the rest of the table metadata is fetched from the filesystem/object storage metadata location.
```

Let me know if this sounds better @hantangwangd

Reply:
Sure, that would be better. Thanks for the clarify.